packages/@vuepress/theme-default/components/SidebarLinks.vue

```diff
@@ -96,7 +96,7 @@ function descendantIsActive (route, item) {
       } else {
         return child.type === 'page' && isActive(route, child.path)
       }
-    })
+    }) || item.path && isActive(route, item.path)
   }
```

Comment:
Sorry for the very late review. Thanks for the PR, and the functionality looks good to me. 

Only a small suggestion, there're too many conditions without comments and variable names for others to understand. Would you like to try the code below

```js
function descendantIsActive (route, item) {
  if (item.type === 'group') {
    const childIsActive = item.path && isActive(route, item.path)
    const grandChildIsActive = item.children.some(child => {
      if (child.type === 'group') {
        return descendantIsActive(route, child)
      } else {
        return child.type === 'page' && isActive(route, child.path)
      }
    })

    return childIsActive || grandChildIsActive
  }
  return false
}
```

Reply:
Thanks, I think that's much clearer. I've implemented your suggestion.